dev-docs/bidders/equativ.md

```diff
@@ -8,19 +8,19 @@ dsa_supported: true
 gvl_id: 45
 usp_supported: true
 coppa_supported: true
-gpp_sids: tcfeu, usp
+gpp_sids: tcfca, tcfeu, usnat, usstate_all, usp
 schain_supported: true
 dchain_support: false
 userIds: all
-media_types: banner
+media_types: banner, video, native
 safeframes_ok: true
 deals_supported: false
 floors_supported: true
-fpd_supported: false
+fpd_supported: true
```

Comment:
You can't claim First Party Data support without explicitly noting the text below what specific FPD fields are supported: site.ext.data, site.content.data, user.data, user.ext.data, imp.ext.data

Reply:
I've added information about FPD.